File: Proyecto/es.efor.PryBase.Angular/webapp/src/app/shared/api/models/menu-dto.ts
/* tslint:disable */
import { FunctionalityDto } from './functionality-dto';
import { ModuleDto } from './module-dto';
export interface MenuDto {
  department?: null | string;
  functionalities?: null | Array<FunctionalityDto>;
  loginDisplayName?: null | string;
  modules?: null | Array<ModuleDto>;
}

File: Proyecto/es.efor.PryBase.Infraestructure/DTO/PermissionsDTOs/MenuDTO.cs
using System.Collections.Generic;

namespace es.efor.PryBase.Infraestructure.DTO.PermissionsDTOs
{
    public class MenuDTO
    {
        #region Constructor
        public MenuDTO()
        {
            this.Modules = new List<ModuleDTO>();
            this.Functionalities = new List<FunctionalityDTO>();
        }
        #endregion

        #region Propiedades 
        public string LoginDisplayName { get; set; }
        public string Department { get; set; }
        public List<ModuleDTO> Modules { get; set; }
        public List<FunctionalityDTO> Functionalities { get; set; }
        #endregion
    }
}
